AODN/AODN-WAVE-NRT/ARDC_SOFAR_API_NRT/config/template_uwa.json

```diff
@@ -0,0 +1,167 @@
+{
+  "_dimensions": {
+    "TIME": null,
+    "DEPTH": 1
+  },
+  "_variables": {
+    "TIME": {
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "units": "days since 1950-01-01 00:00:00 UTC",
+      "calendar": "gregorian",
+      "standard_name": "time",
+      "long_name": "time",
+      "axis": "T",
+      "valid_min": 0.0,
+      "valid_max": 90000.0
+    },
+    "LATITUDE": {
+      "_FillValue": 999999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "standard_name": "latitude",
+      "long_name": "latitude of each observation",
+      "units": "degrees_north",
+      "axis": "Y",
+      "valid_min": -90.0,
+      "valid_max": 90.0,
+      "reference_datum": "WGS84 coordinate reference system"
+    },
+    "LONGITUDE": {
+      "_FillValue": 999999.0,
+      "_datatype": "float64",
```

Comment:
TIME, LATITUDE and LONGITUDE should be double, don't we have to substitute "float64" for double then?

Reply:
float64 is double precision.